File: src/main/java/io/github/oleksivio/telegram/bot/api/model/annotation/validator/impl/LocationFilterValidator.java
package io.github.oleksivio.telegram.bot.api.model.annotation.validator.impl;

import io.github.oleksivio.telegram.bot.api.model.annotation.validator.FilterValidator;
import io.github.oleksivio.telegram.bot.api.model.annotation.validator.CustomValidator;
import io.github.oleksivio.telegram.bot.api.model.objects.std.Location;

public class LocationFilterValidator extends FilterValidator<Location> {

    public LocationFilterValidator(CustomValidator<Location> validator) {
        super(Location.class, validator);
    }
}

File: core/src/main/kotlin/io/github/oleksivio/tl/kbot/core/model/annotation/validator/impl/LocationFilterValidator.kt
package io.github.oleksivio.tl.kbot.core.model.annotation.validator.impl

import io.github.oleksivio.tl.kbot.core.model.annotation.validator.FilterValidator
import io.github.oleksivio.tl.kbot.server.api.objects.std.Location
import io.github.oleksivio.tl.kbot.core.core.controller.handler.check.Validator

class LocationFilterValidator(validator: Validator<Location>) : FilterValidator<Location>(
    Location::class, validator)